libraries/chain/include/graphene/chain/protocol/asset_ops.hpp

```diff
@@ -229,7 +229,7 @@ namespace graphene { namespace chain {
 
       account_id_type fee_payer()const { return account; }
       void            validate()const {
-         FC_ASSERT( amount.amount > 0, "Must settle at least 1 unit" );
+         FC_ASSERT( !"virtual operation" );
       }
```

Comment:
Hm, perhaps this requires a hardfork after all? What if the op turns up in a proposal?

Reply:
Yes, you're correct. I'll revert this change. Let's fix it more carefully in another PR.